### PR TITLE
Unum! The card game.

### PR DESCRIPTION
**Pull request by Burzah**
## What Does This PR Do
Unum! The card game that is certainly not trademarked! Play with your friends as you work to match and discard cards. This is a loose port of https://github.com/BeeStation/BeeStation-Hornet/pull/1422
## Why It's Good For The Game
Adds a game within a game that people may enjoy. Facilitates RP opportunities. 
## Images of changes
image
## Testing
Spawned Unum deck and tested for proper card function including names and sprites.
Checked to see if existing card control functions were maintained.
## Changelog
:cl: Burza and McRamon
add: Added the Unum card deck.
add: Added Unum to loadout menu, arcade prize vendor, labor camp point vendor, merch vendor, and gifts.
/:cl:

### CODE REVIEW

File: code/modules/arcade/prize_datums.dm [hunk 1]
```diff
@@ -141,6 +141,12 @@ GLOBAL_DATUM_INIT(global_prizes, /datum/prizes, new())
 	typepath = /obj/item/clothing/gloves/ring/shadow
 	cost = 40
 
+/datum/prize_item/unum
+	name = "Deck of UNUM! Cards"
+	desc = "Everyone's favorite card game! "
```
Comment (Henri215) [inline code/modules/arcade/prize_datums.dm:146]:
```suggestion
	desc = "Everyone's favorite card game!"
```

Comment (DGamerL) [inline code/modules/arcade/prize_datums.dm:146]:
```suggestion
	desc = "Everyone's favourite card game! "
```
I'm 99% sure we prefer bri'ish over simplified english

File: code/modules/games/unum.dm [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/// A deck of unum cards. Classic.
+/obj/item/deck/unum
+	name = "\improper UNUM! deck"
+	desc = "A deck of unum cards. House rules to argue over not included."
```
Comment (DGamerL) [inline code/modules/games/unum.dm:4]:
```suggestion
	desc = "A deck of UNUM! cards. House rules to argue over not included."
```
Consistent with the majority?

**Review by DGamerL — COMMENTED** [review]
Quick first look, I'm taking another one in a bit

File: code/modules/games/unum.dm (hunk 2 above)
Comment (Burzah) [reply code/modules/games/unum.dm:4]: Oops. I missed that one. I went around and changed it in all the other places.

**Comment by Burzah** [comment]
Sprites have been updated. Special thanks to McRamon for providing them!

**Review by S34NW — APPROVED** [review]

**Comment by spacegeese** [comment]
now we need a card deck rework to tie it all together

**Pull request merged**